lib/src/auth/auth.dart

```diff
@@ -16,12 +16,12 @@
 import 'dart:async';
 import 'dart:convert';
 
-import 'package:googleapis_auth/auth_io.dart' as auth;
-import 'package:googleapis_auth/src/crypto/rsa_sign.dart';
+import 'package:googleapis_auth/auth_browser.dart' as auth;
 import 'package:grpc/src/shared/status.dart';
```

Comment:
This import still imports from the local package using a `package:` URI. I can see you fixed all the rest :)

Reply:
Thanks!
Fixed